### DISCUSSION TITLE
How to set full width for landing page only?

### DISCUSSION THREAD

**Discussion by rajk-cerebras** [executablebooks/meta#317] [Q&A]
Hey folks, 

I am wondering if it is possible, with the Sphinx Book Theme, to set only the home page with no right sidebar, like here: https://plaid.com/docs/. I really like the left sidebar of the Sphinx Book theme where the documentation sections are displayed right there. However, I use raw html to create panels on the homepage, but these panels do not extend to the right sidebar (see the attached screenshot below).   

![image](https://user-images.githubusercontent.com/74557030/114640112-45776200-9c84-11eb-8a0b-4359bdf0ec6c.png)

Does anyone have suggestions on how I can remove the right sidebar only for the homepage? Thanks!

**Comment by choldgraf** [comment]
Could you try wrapping them in a div with a full-width class? That is how notes etc are expanded to fit the page

**Reply by rajk-cerebras** [reply]
Yeah, that's an idea. I will update this thread when I tried that, thanks!

**Comment by rajk-cerebras** [comment]
It worked like a charm, thanks, @choldgraf. See the working example snippet below. I had to place the `<h2>` tag inside the raw html so that it will not generate the right sidebar entry. 

```
.. raw:: html

  <div class="full-width">

   <h2>Get Started</h2>

   <div class="card-deck">

    <div class="card">

```
